File: stories/examples/py/manage-customer-orders/track-order-status/send-shipment-notification/send_shipment_notification_stories.py
"""Story data — regeneratable. Do not add logic or imports."""

from __future__ import annotations

from typing import Final


SEND_SHIPMENT_NOTIFICATION: Final = {
    "story":        "Send Shipment Notification",
    "actor":        "System",
    "domain_terms": ("Shipment", "Shipment Notification", "Tracking Number", "Notification Channel"),
    "evidence":     (),
}

File: _retired/stories/examples/tsx/manage-customer-orders/track-order-status/send-shipment-notification/send-shipment-notification-stories.ts
import type { Story } from '../../story-types'

export const SendShipmentNotification = {
  story:       'Send Shipment Notification',
  actor:       'System',
  domainTerms: ['Shipment', 'Shipment Notification', 'Tracking Number', 'Notification Channel'],
  evidence:    [],

} as const satisfies Story
